App/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,109 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+    // switch to true to log data before validating 
+    private static final boolean DEBUGGING = true;
+
+    private void serverLog(
+        String name, String gender, Integer age, Boolean medicalAssistance, Integer numberOfFamilyMembers, 
```

Comment:
Yikes, 7 arguments to a function... Leave it as-is for now, but you'll most likely want to change this to just accept a User object once that exists. (Or Cameron could steal this and add it as the toString method for User - are y'all familiar with that trick?)

Reply:
Let's just say I lost a couple of quarters doing that but I didn't know a better way aside from making a class. Once User is all set up then I'll definitely be using that instead. 

The toString would actually be awesome and make this file so much more readable and focused. A huge +1 to that idea 🙂

Reply:
Oh I mean it's perfectly fine to do this for now, it's the best option I can think of without making a class or similar. I was mainly pointing this out to show one reason why @cpayton-source 's classes are so helpful to have. :)